src/lib/styles/global/button.scss

```diff
@@ -1,12 +1,17 @@
 @use "../mixins/effect";
 @use "../mixins/fonts";
+@use "../mixins/button";
```

Comment:
i'd rather to a class called .button that can be set on a link than a mixin.

Reply:
That's a good point.

I believe that supporting all the other button types would be a bit more work than what I was expecting in this PR.

Reply:
But like this you are actively preventing from it being used anywhere else + the code is there twice, once in the button css and one in tha a.button css.